sympy/stats/crv_types.py

```diff
@@ -3444,6 +3447,111 @@ def Pareto(name, xm, alpha):
 
     return rv(name, ParetoDistribution, (xm, alpha))
 
+
+#-------------------------------------------------------------------------------
+# Pert distribution ------------------------------------------------------------
+
+
+class PERTDistribution(SingleContinuousDistribution):
+    _argnames = ('a', 'b', 'c')
+
+    @property
+    def set(self):
+        return Interval(self.a, self.c)
+
+    @staticmethod
+    def check(a, b, c):
+        _value_check(b > a, "Parameter b must be greater than a.")
+        _value_check(c > b, "Parameter c must be greater than b.")
+        _value_check(c > a, "Parameter c must be greater than a.")
+
+    def pdf(self, x):
+        a, b, c = self.a, self.b, self.c
+        alpha = 1 + 4*(b - a/(c - a))
+        beta = 1 + 4*(c - b/(c - a))
+        num = (x - a)**(alpha - 1)*(c - x)**(beta - 1)
+        den = beta_fn(alpha, beta)*(c - a)**(alpha + beta - 1)
+        return num/den
+
+    def _cdf(self, x):
+        a, b, c = self.a, self.b, self.c
+        alpha = 1 + 4*(b - a/(c - a))
+        beta = 1 + 4*(c - b/(c - a))
+        z = (x - a)/(c - a)
+        return betainc_regularized(alpha, beta, 0, z)
+
+    def expectation(self, expr, var, **kwargs):
+        a, b, c = self.a, self.b, self.c
+        return (a + 4*b + c)/6
```

Comment:
We should generally avoid overriding `expectation` and let sympy calculate on its own. See the issue due to which the PR in OP was stalled: https://github.com/sympy/sympy/issues/16983

Reply:
You can actually check that by removing these 3 lines and see if it is able to find the expectation.

Reply:
No, sympy is unable to find expectation automatically. Should i close this PR then?

Reply:
Well, if SymPy is unable to calculate `expectation` automatically, then that's a bit non-ideal. Adding a method to directly obtain `expectation` in the PERT's distribution class is fine by me. 
@Smit-create What do you think?